packages/SwingSet/src/kernel/vat-loader/transcript.js

```diff
@@ -1,5 +1,21 @@
 import djson from '../../lib/djson.js';
 
+// Indicate that a syscall is missing from the transcript but is safe to
+// perform during replay
+const missingSyscall = Symbol('missing transcript syscall');
```

Comment:
Any particular reason to use a Symbol rather than just a string? Or an empty object?

Reply:
Strings are more forgeable and I have a habit of using unique symbol for sentinel values.